server/boot/index.js

```diff
@@ -38,6 +39,23 @@ function setup() {
 
 		// setup logger
 		app.use( morgan( 'dev' ) );
+
+		if ( config.isEnabled( 'wpcom-user-bootstrap' ) ) {
```

Comment:
It looks good but I'm worried it might have some side effects which might force us to revert. Probably would have been better in a new PR imo

Reply:
I agree, after we finish the PR I think we should drop this commit before merging

Reply:
You are right, https://github.com/Automattic/wp-calypso/pull/18264